src/main/java/org/ballerinalang/docgen/docs/html/HtmlDocumentWriter.java

```diff
@@ -91,14 +96,33 @@ public void write(Collection<BLangPackage> packages) throws IOException {
         if (BallerinaDocUtils.isDebugEnabled()) {
             out.println("Generating HTML API documentation...");
         }
-        Files.createDirectories(Paths.get(outputFilePath));
```

Comment:
@imesh hmm.... it shows as this PR removes that line?? 
<img width="994" alt="screen shot 2017-02-22 at 11 02 32 am" src="https://cloud.githubusercontent.com/assets/413016/23198570/7e5d2192-f8ee-11e6-8c3c-11898e2bb493.png">